src/configuration/services/mongodb.md

```diff
@@ -1,32 +1,32 @@
 # MongoDB (Database service)
 
-MongoDB is a cross-platform document-oriented database.
-
-See the [MongoDB documentation](https://docs.mongodb.com/manual/) for more information.
+MongoDB is a cross-platform, document-oriented database.
 
 ## Supported versions
 
 * 3.0
 
-## Relationship
+## Relationships
```

Comment:
It's `Relationship` in every other service page.  Again, consistency.  Changing the pattern can be discussed elsewhere.